dbms/src/Storages/DeltaMerge/SSTFilesToDTFilesOutputStream.cpp

```diff
@@ -124,21 +124,8 @@ bool SSTFilesToDTFilesOutputStream<ChildStream>::newDTFileStream()
         return false;
     }
 
-    DMFileBlockOutputStream::Flags flags{};
-    switch (method)
```

Comment:
After removing these lines, the SSTFilesToDTFilesOutputStream::method is a useless private member and should be remove